docs/tutorials/multimodal/image_prediction/beginner_image_cls.md

```diff
@@ -15,9 +15,24 @@ We can load a dataset by downloading a url data automatically:
 ```{.python .input}
 import warnings
 warnings.filterwarnings('ignore')
-from autogluon.vision import ImageDataset
-train_dataset, _, test_dataset = ImageDataset.from_folders("https://autogluon.s3.amazonaws.com/datasets/shopee-iet.zip")
```

Comment:
The HPO tutorial also has this import. Could you also fix it?

Reply:
Yeah, sure. Will include that as well, this is just to test it first